pkg/eks/api.go

```diff
@@ -113,6 +122,8 @@ func New(spec *api.ProviderConfig, clusterSpec *api.ClusterConfig) *ClusterProvi
 	provider.cfn = cloudformation.New(s)
 	provider.eks = awseks.New(s)
 	provider.ec2 = ec2.New(s)
+	provider.elb = elb.New(s)
+	provider.elbv2 = elbv2.New(s)
```

Comment:
Please also see custom endpoint logic below, we need to add `AWS_ELB_ENDPOINT` and `AWS_ELBV2_ENDPOINT` for consistency.

https://github.com/weaveworks/eksctl/pull/1010/files#diff-eb5619b1d325aef63a641b639f6dde9fL132